### ISSUE TITLE
Fix hyper-parameters

### ISSUE THREAD

**Issue by Geoc2022** [AzureCoral/across-the-spiderverse-style-transfer#2]
Add learning rate to hyper-parameters with a good value.

Also look into this: Currently, hyper-parameters are set up in the init of StyleTransfer but tbh they don't really get used until train is called. So it maybe more efficient to initialize them in train, which would make it easier to search for good hyper-parameters. Find some example which does this.

```
class StyleTransfer():
  def __init__(self, style_layers: List[str], 
               content_layers: List[str], 
               style_images: List[tf.Tensor], 
               content_image: tf.Tensor, 
               style_weight: float = 1e-2,
               content_weight: float = 1e4,
               total_variation_weight: float = 100):
    """
    Initializes the StyleTransfer.

    Parameters:
    style_layers (List[str]): The names of the style layers to include in the model.
    content_layers (List[str]): The names of the content layers to include in the model.
    style_image (tf.Tensor): The style image tensor.
    content_image (tf.Tensor): The content image tensor.
    style_weight (float): The weight of the style loss.
    content_weight (float): The weight of the content loss.
    
    """
```
